### ISSUE TITLE
No mhi-first if previous video is grayscale (1 plane) 

### ISSUE THREAD

**Issue by aleksati** [balintlaczko/VideoAnalysis#109]
The bug occurs in the [jit.op @ op +] in the [VA_motion_image_first], only when a previous video has 1-plane and the current video has 4-planes. So the [jit.op @ op +] maybe needs to be explicitly told to adapt to the current matrix count.

**Comment by aleksati** [comment]
This is working. Will push the updated [VA_motion_image_first] soon
EDIT: spoke a little too soon here. Still issues here with the jit.op and going from 1-to-4 planes in the export.

**Comment by aleksati** [comment]
Okay, I'm pretty sure I discovered the issue here. 

My theory is that the [jit.op] adapts to the left inlet matrix stream, although I didn't find any documentation on this. But it makes sense because when a 4-plane movie is exporting after(!) a movie with 1-plane, its first-frame comes into the [jit.op] first with a different planecount than what the [jit.op] is currently operating on. Therefore the video comes out completely white or black. 

When I changed it so that the [jit.op @ op +] receives the matrix (left inlet) just before the first-frame in every case, it seems to work just fine.
